#####################################
##### Road following parameters #####
#####################################

model_type: 'iisc_rf' # 'iisc_rf', 'giusti_rf', 'dronet_rf'
gost_thresh: 40.0
lr_bias: True
dirn_history_sz: 20
dirn_history_thresh: 17
confusion_correction: True
bias_lr_thresh: 5.0
confusion_corr_thresh1: 20.0
confusion_corr_thresh2: 25.0
